temel_bilgiler/giriş/motivasyon_ve_dusunce_yapisi.md

```diff
@@ -0,0 +1,171 @@
+1. Araştırma, problem çözme ve eleştirel düşünme pratiği olmadan, mülakatlar zor geçebilir çünkü başvuran kişilerin yapay zeka araçlarını kullanmalarına izin verilmeyeceği çok muhtemeldir.
+
+Öğreniminiz için yapay zeka araçlarını kullanmanızı önermiyoruz.
+
+### Çalışma Saatlerinizi Yönetmek
+
+Odin üzerinde haftada bir kez çalışmak yerine **sürekli** zaman ayırırsanız daha başarılı olabilirsiniz. Her gün belirli bir saatte ve belirli bir hedefle çalışma alışkanlığı edinmek, tutarlı bir ilerleme kaydetmenizi sağlayacaktır.
+
+Kavramları anlamanız diğer insanlardan daha uzun sürebilir ya da daha az zamanınızı alabilir. Bu, diğerlerinden daha zeki olduğunuz veya olmadığınız anlamına gelmez. Bu farklı deneyimleriniz olduğu anlamına gelir. Bir mühendisin yanında büyüyen biri, büyümeyen birine göre bazı avantajlara sahip olabilir, ancak bu o becerileri öğrenemeyeceğiniz anlamına gelmediğini unutmamalıyız.
+
+Odin Projesi kolej veya üniversite gibi değildir, kendi kendine ilerler ve devam etmeden önce kavramları gerçekten sağlam bir şekilde kavramanıza olanak tanır. Okulda, ayak uydurmaya zorlanırsınız ya da başarısız olursunuz. Burada farklı olan şey, Odin Projesi'ne girerken çok fazla bilgiye sahip olmanız beklenmiyor; ön koşul yok. Sadece bilgisayarında e-postalarını nasıl kontrol edeceklerini bilen insanların buraya gelerek başarılı olduklarını gördük. Odin Projesi'ni statik bir zaman çizelgesi gibi ele almanız anlaşılabilir, ancak bu yanlış beklentinin bir işaretidir. Henüz neyi bilmediğinizi bilmiyorsanız bu tamamen normal! Odin Projesi'nde hiçbir şey için son tarih yok, bu yüzden doğru yapmak ve konuları tartışmak için istediğiniz kadar zaman harcayabilirsiniz.
+
+Son teslim tarihleri (Deadlines) gereksiz strese neden olur. Odin Projesi ücretsiz ve açık bir platform olduğundan, bu müfredatta son teslim tarihi gibi bir şeye bağlı olmadığınızı belirtmek isteriz. Ancak kendi teslim tarihlerinizi belirlemek, aceleye getirilmemesi gereken kavramları aceleye getirmek için iyi bir yoldur. Bu kurs daha çok araştırma tabanlıdır. Yani görevleri ve projeleri tamamlamak için araştırma yapmanız gerekecektir. Doğru şekilde size yardımcı olacak makaleyi veya gönderiyi ne kadar hızlı bulabileceğiniz veya kendinize teslim tarihi belirlediyseniz teslim tarihlerinize yetişebileceğiniz belli olmayabilir, ancak müfredat boyunca çok şey öğreneceğinize eminiz. Bu tür araştırmalar yapan ve daha iyi çözümler yazmak için çabalayan insanlar gelecekte daha iyi geliştiriciler olma eğilimindedir. Cevaplarınızı bulmak için bir şeyleri nasıl sorgulayacağınızı öğrenmenizin ne kadar süreceğini bilemezsiniz. Bu konuda sağlam bir kılavuz yoktur. Odin Projesi'ni _hemen şimdi_ yüksek maaşlı bir işe ihtiyacınız olduğu için yapıyorsanız, belirlediğiniz zaman dilimi içinde sağlam bir geliştirici olmayacaksınız. Stres ve kaygı da öğrenmenize yardımcı olmayacaktır. Sadece rahatlayın ve bu yolculuğun tadını çıkarın.
```

Comment:
`Ancak kendi teslim tarihlerinizi belirlemek, aceleye getirilmemesi gereken kavramları aceleye getirmek için iyi bir yoldur.` cumlesini dogru cevirdigimden emin degilim

Reply:
Bence paragrafların genel anlamından ötürü "should not" olduğu için cümle olumsuz olmalı. Ben böyle çevirirdim açıkçası:
"Ancak kendi teslim tarihlerinizi oluşturmak, aceleye getirilmemesi gereken kavramları aceleye getirmenize neden olur."

Reply:
Onerin icin tesekkurler, bununla ilgili bir commit atacagim.

Reply:
> `Ancak kendi teslim tarihlerinizi belirlemek, aceleye getirilmemesi gereken kavramları aceleye getirmek için iyi bir yoldur.` cumlesini dogru cevirdigimden emin degilim

`Kendi teslim tarihlerinizi oluşturmak da aceleye getirilmemesi gereken kavramları aceleye getirmenize neden olabilir.` olarak guncelledim